gptme/message.py

```diff
@@ -211,6 +215,17 @@ def format_msgs(
         if highlight:
             color = ROLE_COLOR[msg.role]
             userprefix = f"[bold {color}]{userprefix}[/bold {color}]"
+            
+        # Add status emoji for system messages
+        if msg.role == "system" and msg.status:
+            status_emoji = {
+                "success": "✅",
+                "error": "❌",
+                "info": "ℹ️",
+            }
+            if emoji := status_emoji.get(msg.status):
+                userprefix = f"{userprefix} {emoji}"
```

Comment:
In gptme-webui I simply check the first line with:

```py
first_line = msg.content.split("\n", 1)[0].lower()
isSuccess = first_line.startswith(["saved", "success", "patch successfully"])
isError = first_line.startswith(["error", "failed"])
```

Since it's just a design thing, I think such a lightweight approach makes more sense.
The "info" emoji also doesn't make much sense to me (how is it different from no status?)

Reply:
> The "info" emoji also doesn't make much sense to me (how is it different from no status?)

The info emoji was suggested by gptme yeah, I should just scrap it.